common/domain/attrValidator.go

```diff
@@ -184,3 +184,29 @@ func (d *AttrValidatorImpl) validateClusterName(
 	}
 	return nil
 }
+
+func (d *AttrValidatorImpl) validateActiveActiveDomainReplicationConfig(
```

Comment:
Where is it used?

Reply:
I screwed up and forgot to invoke it, fixed.